easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2021.10-intel-2021b.eb

```diff
@@ -0,0 +1,84 @@
+easyblock = 'PythonBundle'
+
+name = 'SciPy-bundle'
+version = '2021.10'
+
+homepage = 'https://python.org/'
+description = "Bundle of Python packages for scientific software"
+
+toolchain = {'name': 'intel', 'version': '2021b'}
+toolchainopts = {'pic': True, 'lowopt': True}
+
+builddependencies = [
+    ('hypothesis', '6.14.6'),
+    ('UnZip', '6.0'),
+]
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('pybind11', '2.7.1'),  # required by scipy
+]
+
+use_pip = True
+
+# order is important!
+exts_list = [
+    ('numpy', '1.21.3', {
+        'sources': ['%(name)s-%(version)s.zip'],
+        'patches': [
+            'numpy-1.18.2-mkl.patch',
+            'numpy-1.20.3_disable-broken-override-test.patch',
+            'numpy-1.20.3_disable_fortran_callback_test.patch',
+        ],
+        'checksums': [
+            '63571bb7897a584ca3249c86dd01c10bcb5fe4296e3568b2e9c1a55356b6410e',  # numpy-1.21.3.zip
+            'ea25ad5c0148c1398d282f0424e642fb9815a1a80f4512659b018e2adc378bcf',  # numpy-1.18.2-mkl.patch
+            # numpy-1.20.3_disable-broken-override-test.patch
+            '43cc2e675c52db1776efcc6c84ebd5fc008b48e6355c81087420d5e790e4af9b',
+            # numpy-1.20.3_disable_fortran_callback_test.patch
+            '44975a944544fd0e771b7e63c32590d257a3713070f8f7fdf60105dc516f1d75',
+        ],
+    }),
+    ('ply', '3.11', {
+        'checksums': ['00c7c1aaa88358b9c765b6d3000c6eec0ba42abca5351b095321aef446081da3'],
+    }),
+    ('gast', '0.5.2', {
+        'checksums': ['f81fcefa8b982624a31c9e4ec7761325a88a0eba60d36d1da90e47f8fe3c67f7'],
+    }),
+    ('beniget', '0.4.1', {
+        'checksums': ['75554b3b8ad0553ce2f607627dad3d95c60c441189875b98e097528f8e23ac0c'],
+    }),
+    ('pythran', '0.10.0', {
+        'checksums': ['9dac8e1d50f33d4676003e350b1f0c878ce113e6f907920e92dc103352cac5bf'],
+    }),
```

Comment:
If scipy fails to build with pythran when using intel compilers, should we be removing these items that were only added to support building scipy with pythran? Though that would then make this inconsistent with the foss version.

Reply:
Yeah, wasn't sure here either...

It's not totally broken though, Pythran itself seems functional (cfr. https://github.com/serge-sans-paille/pythran#basic-usage):

```shell
$ module load SciPy-bundle/2021.10-intel-2021b

$ pythran dprod.py

$ ls dprod.*
dprod.cpython-39-x86_64-linux-gnu.so  dprod.py

$ python -c 'import dprod; print(dprod.__file__); print(dprod.dprod(list(range(1, 1000)), list(range(1001, 2000))))'
/tmp/dprod.cpython-39-x86_64-linux-gnu.so
832333500
```

So I'd rather leave it in, perhaps a patch turns out that we can apply easily to build `scipy` with Pythran support later...